recipe/meta.yaml

```diff
@@ -22,7 +22,6 @@ requirements:
   run:
```

Comment:
There are missing dependencies in host: setuptools and wheel. We should explicitly use them